<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtkRangeWidget</class>
 <widget class="QWidget" name="VtkRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="scalarLayout">
     <item>
      <widget class="QLabel" name="scalarLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Scalar Value: 99999 to 99999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scalarButton">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
